src/index.html

```diff
@@ -15,8 +15,39 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="container">
+        <a
+          href="https://th.bing.com/th/id/OIP.axjC06-qxFXElNPCbM1MPAAAAA?rs=1&pid=ImgDetMain"
+          target="_blank"
+        >
```

Comment:
Keep your attributes correctly formatted. Since the `a` tag has more than two attributes and some with long values, each attribute should start on a new line with a 2-space indentation related to the tag, and the closing bracket should align with the opening tag.